src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> convertedTransaction = new ArrayList<>();
+        for (int i = 1; i < data.size(); i++) { // Skip header
+            String[] parts = data.get(i).split(",");
+            FruitTransaction.Operation operation = mapOperation(parts[0]);
+            String fruit = parts[1];
+            int quantity = Integer.parseInt(parts[2]);
```

Comment:
Consider adding error handling for `Integer.parseInt` to catch `NumberFormatException` in case the quantity is not a valid integer.